clang/test/SemaSYCL/esimd-private-statics.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -pedantic %s
+// expected-no-diagnostics
+
+// This test checks that non-const statics are allowed for ESIMD
+
+static __attribute__((opencl_private)) __attribute__((sycl_explicit_simd)) int esimdPrivStatic;
```

Comment:
Can you also add a test that this works (or fails, if that's expected) on static data members?

Reply:
Done